planning_tests/utility/correct_output.py

```diff
@@ -0,0 +1,29 @@
+def correct(path):
```

Comment:
This function is not currently used. It will be used to correct the plan generated by UP before calling the validate function when it will be implemented